recipes/ccache-toolchain/activate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export _OLD_PATH=${PATH}
```

Comment:
I thought the activate and deactivate happened in a way such that the last activate is deactivated first. Let me check that